recipes/moltenvk/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
   "1.1.1":
     folder: all
+  "1.1.0":
```

Comment:
why are you adding an older version?

Reply:
https://github.com/conan-io/conan-center-index/pull/4762#issue-583194666

Reply:
Thanks I skipped over that.